README.md

```diff
@@ -149,7 +150,7 @@ need to hack the setup the same way we do for testing:
 For more information on RBAC you can check [CloudAPI docs][cloudapi] and
 the [Access Control User Guide][acuguide].
 
-[cloudapi]: https://apidocs.joyent.com/cloudapi/
+[cloudapi]: https://apidocs.tritondatacenter.com/cloudapi/
 [acuguide]: https://docs.joyent.com/jpc/rbac
```

Comment:
Maybe this link can be replaced with: https://docs.tritondatacenter.com/public-cloud/rbac

Reply:
Also fixed